### PR TITLE
Implement CLOSED_OPEN and TIME filters for SQL to allow proper filtering

### PR DESCRIPTION
**Pull request by Clebal**

### CODE REVIEW

File: packages/react-core/__tests__/filters/FilterQueryBuilder.test.js [hunk 1]
```diff
@@ -46,10 +46,24 @@ describe('Filters to SQL', () => {
...
+      column5: {
+        closed_open: {
+          owner: 'widgetId3',
+          values: [[0, 1]]
+        }
       }
     };
     expect(filtersToSQL(customFilters)).toEqual(
-      "WHERE (column1 in('a','b','c')) and ((column2 >= 1  and  column2 < 2)) and (column3 in(1,2,3))"
+      "WHERE (column1 in('a','b','c')) and ((column2 >= 1  and  column2 <= 2)) and (column3 in(1,2,3)) and ((cast(column4 as timestamp) >= cast('1970-01-01T00:00:00.000Z' as timestamp)  and  cast(column4 as timestamp) <= cast('1970-01-01T00:00:00.001Z' as timestamp))) and ((  column5 < 1))"
```
Comment (VictorVelarde) [inline packages/react-core/__tests__/filters/FilterQueryBuilder.test.js:66]: Maybe I'm wrong, but shouldn't it be `((  column5 >= 0 and column5 < 1))` ?

Comment (Clebal) [reply packages/react-core/__tests__/filters/FilterQueryBuilder.test.js:66]: YES! Nullish values are 😠

**Review by eamador — APPROVED** [review]
just the Victor's comment, but looks great! 🚀

File: packages/react-core/src/filters/FilterQueryBuilder.js [hunk 2]
```diff
@@ -17,39 +17,68 @@ export const getApplicableFilters = (filters = {}, owner) => {
...
+        }`
+    );
+
+    return joinFilters(queryFilters);
+  },
+  [FilterTypes.CLOSED_OPEN](column, filterValues) {
+    const queryFilters = filterValues.map(
+      ([left, right]) =>
+        `${left ? `${column} >= ${left}` : ``} ${left && right ? ' and ' : ''} ${
+          right ? `${column} < ${right}` : ``
```
Comment (VictorVelarde) [inline packages/react-core/src/filters/FilterQueryBuilder.js:56]: I would say we have an issue with 0 treated as a falsy value vs undefined

Comment (Clebal) [reply packages/react-core/src/filters/FilterQueryBuilder.js:56]: Yes, i'm fixing it

**Comment by Clebal** [comment]
Fixed @eamador @VictorVelarde

**Review by VictorVelarde — APPROVED** [review]
👍🏻

**Pull request merged**